cypress/e2e/repository/structure/activity-deletion.cy.js

```diff
@@ -0,0 +1,60 @@
+import {
+  createChildActivity,
+  createRootActivity,
+  findActivityItem,
+  generateActivityName
+} from './utils';
+
+const ROOT_ACTIVIY_TYPES = ['Module', 'Page'];
+const CHILD_ACTIVITY_TYPES = ['Module', 'Lesson', 'Knowledge check', 'Page'];
+
+const createdParent = [];
+const createdChild = [];
```

Comment:
We need to replace this. Since it has some complexity, I'll do a separate PR or you can replace it with the seed that we created (needs to be done via alias)